mlir/test/Dialect/Affine/access-analysis.mlir

```diff
@@ -0,0 +1,67 @@
+// RUN: mlir-opt %s -split-input-file -test-affine-access-analysis -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL: func @loop_1d
+func.func @loop_1d(%A : memref<?x?xf32>, %B : memref<?x?x?xf32>) {
+   %c0 = arith.constant 0 : index
+   %M = memref.dim %A, %c0 : memref<?x?xf32>
+   affine.for %i = 0 to %M {
+     affine.for %j = 0 to %M {
+       affine.load %A[%c0, %i] : memref<?x?xf32>
+       // expected-remark@-1 {{contiguous along loop 0}}
```

Comment:
Nit: here and below, it is possible to use `// expected-remark @above {{..}}` to reduce the brittleness of manual line offset calculation. This construct works across multiple `expected-remark` directives, and potentially other comments.

Reply:
Done.